app/adminPage.html

```diff
@@ -163,7 +163,7 @@ <h4 class="search-heading">Filter results</h4>
                                 </div>
                             </div>
                     </div>
-                    <table class="table">
+                    <table id="main-table" class="table">
```

Comment:
you are not allowed `-` in IDs and `main-table` isn't a great ID anyway, why not something like `applicantTable`